### PR TITLE
Add compat data for SharedWorkerGlobalScope (Worker API)

### PR DESCRIPTION
**Pull request by maboa**
https://developer.mozilla.org/en-US/docs/Web/API/SharedWorkerGlobalScope

### CODE REVIEW

File: api/SharedWorkerGlobalScope.json [hunk 1]
```diff
@@ -0,0 +1,287 @@
+{
+  "api": {
+    "SharedWorkerGlobalScope": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/SharedWorkerGlobalScope",
+        "support": {
+          "webview_android": {
+            "version_added": false
```
Comment (jpmedley) [inline api/SharedWorkerGlobalScope.json]: Please change to `true`.

File: api/SharedWorkerGlobalScope.json [hunk 2]
```diff
@@ -0,0 +1,287 @@
...
+          },
+          "chrome": {
+            "version_added": "4"
+          },
+          "chrome_android": {
+            "version_added": false
```
Comment (jpmedley) [inline api/SharedWorkerGlobalScope.json]: `true`

File: api/SharedWorkerGlobalScope.json [hunk 3]
```diff
@@ -0,0 +1,287 @@
...
+          "description": "<code>open</code>",
+          "support": {
+            "webview_android": {
+              "version_added": null
+            },
+            "chrome": {
+              "version_added": true
+            },
+            "chrome_android": {
+              "version_added": true
```
Comment (jpmedley) [inline api/SharedWorkerGlobalScope.json]: All 3 versions of Chrome should be `false`.

File: api/SharedWorkerGlobalScope.json [hunk 4]
```diff
@@ -0,0 +1,287 @@
...
+              "version_added": "55"
+            },
+            "ie": {
+              "version_added": null
+            },
+            "opera": {
+              "version_added": null
+            },
+            "opera_android": {
+              "version_added": true
```
Comment (jpmedley) [inline api/SharedWorkerGlobalScope.json]: Both versions of Opera should be `false`.

File: api/SharedWorkerGlobalScope.json [hunk 5]
```diff
@@ -0,0 +1,287 @@
...
+            }
+          }
+        }
+      },
+      "applicationCache": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/WorkerGlobalScope/applicationCache",
+          "support": {
+            "webview_android": {
+              "version_added": null
```
Comment (jpmedley) [inline api/SharedWorkerGlobalScope.json]: true

File: api/SharedWorkerGlobalScope.json [hunk 6]
```diff
@@ -0,0 +1,287 @@
...
+            },
+            "chrome": {
+              "version_added": "3"
+            },
+            "chrome_android": {
+              "version_added": false
```
Comment (jpmedley) [inline api/SharedWorkerGlobalScope.json]: true

File: api/SharedWorkerGlobalScope.json [hunk 7]
```diff
@@ -0,0 +1,287 @@
...
+              "version_added": "29"
+            },
+            "ie": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": "10.6"
+            },
+            "opera_android": {
+              "version_added": null
```
Comment (jpmedley) [inline api/SharedWorkerGlobalScope.json]: true

File: api/SharedWorkerGlobalScope.json [hunk 8]
```diff
@@ -0,0 +1,287 @@
...
+            "deprecated": false
+          }
+        }
+      },
+      "name": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/WorkerGlobalScope/name",
+          "support": {
+            "webview_android": {
+              "version_added": null
```
Comment (jpmedley) [inline api/SharedWorkerGlobalScope.json]: true

File: api/SharedWorkerGlobalScope.json [hunk 9]
```diff
@@ -0,0 +1,287 @@
...
+              "version_added": "55"
+            },
+            "ie": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": "10.6"
+            },
+            "opera_android": {
+              "version_added": null
```
Comment (jpmedley) [inline api/SharedWorkerGlobalScope.json]: true

File: api/SharedWorkerGlobalScope.json [hunk 10]
```diff
@@ -0,0 +1,287 @@
...
+            "deprecated": false
+          }
+        }
+      },
+      "onconnect": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/WorkerGlobalScope/onconnect",
+          "support": {
+            "webview_android": {
+              "version_added": null
```
Comment (jpmedley) [inline api/SharedWorkerGlobalScope.json]: true

File: api/SharedWorkerGlobalScope.json [hunk 11]
```diff
@@ -0,0 +1,287 @@
...
+              "version_added": "29"
+            },
+            "ie": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": "10.6"
+            },
+            "opera_android": {
+              "version_added": null
```
Comment (jpmedley) [inline api/SharedWorkerGlobalScope.json]: true

File: api/SharedWorkerGlobalScope.json [hunk 12]
```diff
@@ -0,0 +1,287 @@
...
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/WorkerGlobalScope/close",
+          "support": {
+            "webview_android": {
+              "version_added": null
+            },
+            "chrome": {
+              "version_added": null
+            },
+            "chrome_android": {
+              "version_added": null
```
Comment (jpmedley) [inline api/SharedWorkerGlobalScope.json]: false for all 3. (In Chrome's implementation, close() is actually defined on derived interfaces, SharedWorkerGlobalScope to be specific.)

File: api/SharedWorkerGlobalScope.json [hunk 13]
```diff
@@ -0,0 +1,287 @@
...
+              "version_added": "54"
+            },
+            "ie": {
+              "version_added": null
+            },
+            "opera": {
+              "version_added": null
+            },
+            "opera_android": {
+              "version_added": null
```
Comment (jpmedley) [inline api/SharedWorkerGlobalScope.json]: false

Same reason as for Chrome.

File: api/SharedWorkerGlobalScope.json (hunk 1 above)
Comment (maboa) [reply api/SharedWorkerGlobalScope.json]: 👍

File: api/SharedWorkerGlobalScope.json (hunk 2 above)
Comment (maboa) [reply api/SharedWorkerGlobalScope.json]: 👍

File: api/SharedWorkerGlobalScope.json (hunk 3 above)
Comment (maboa) [reply api/SharedWorkerGlobalScope.json]: 👍

File: api/SharedWorkerGlobalScope.json (hunk 4 above)
Comment (maboa) [reply api/SharedWorkerGlobalScope.json]: 👍

File: api/SharedWorkerGlobalScope.json (hunk 5 above)
Comment (maboa) [reply api/SharedWorkerGlobalScope.json]: 👍

File: api/SharedWorkerGlobalScope.json (hunk 6 above)
Comment (maboa) [reply api/SharedWorkerGlobalScope.json]: 👍

File: api/SharedWorkerGlobalScope.json (hunk 7 above)
Comment (maboa) [reply api/SharedWorkerGlobalScope.json]: 👍

File: api/SharedWorkerGlobalScope.json (hunk 8 above)
Comment (maboa) [reply api/SharedWorkerGlobalScope.json]: 👍

File: api/SharedWorkerGlobalScope.json (hunk 9 above)
Comment (maboa) [reply api/SharedWorkerGlobalScope.json]: 👍

File: api/SharedWorkerGlobalScope.json (hunk 10 above)
Comment (maboa) [reply api/SharedWorkerGlobalScope.json]: 👍

File: api/SharedWorkerGlobalScope.json (hunk 11 above)
Comment (maboa) [reply api/SharedWorkerGlobalScope.json]: 👍

File: api/SharedWorkerGlobalScope.json (hunk 12 above)
Comment (maboa) [reply api/SharedWorkerGlobalScope.json]: 👍

File: api/SharedWorkerGlobalScope.json [hunk 14]
```diff
@@ -0,0 +1,287 @@
...
+            "version_added": "7.1"
+          }
+        },
+        "status": {
+          "experimental": false,
+          "standard_track": true,
+          "deprecated": false
+        }
+      },
+      "open": {
```
Comment (Elchi3) [inline api/SharedWorkerGlobalScope.json]: I don't understand what `open` is or where it comes from. @chrisdavidmills you've added this to https://developer.mozilla.org/en-US/docs/Web/API/SharedWorkerGlobalScope#Browser_compatibility, what is it?

Comment (chrisdavidmills) [reply api/SharedWorkerGlobalScope.json]: @Elchi3 this is an interesting mystery — I can't find any record of this ```open``` property anywhere, and running ```console.log(self.open)``` inside a shared worker returns ```undefined```. I wonder if maybe it was added at some point, but then removed from the spec again? @jpmedley, any thoughts on this?

Comment (jpmedley) [reply api/SharedWorkerGlobalScope.json]: It's a mystery to me too.

Comment (Elchi3) [reply api/SharedWorkerGlobalScope.json]: Let's remove this then please.

Comment (chrisdavidmills) [reply api/SharedWorkerGlobalScope.json]: Yup, agreed.

Comment (maboa) [reply api/SharedWorkerGlobalScope.json]: OK. Removed :)

**Pull request merged**